packages/puppeteer-core/src/common/NetworkManager.test.ts

```diff
@@ -692,14 +692,11 @@ describe('NetworkManager', () => {
 
     const finishedRequests: HTTPRequest[] = [];
     const pendingRequests: HTTPRequest[] = [];
-    manager.on(
-      NetworkManagerEmittedEvents.RequestFinished,
-      (request: HTTPRequest) => {
-        finishedRequests.push(request);
-      }
-    );
+    manager.on(NetworkManagerEvent.RequestFinished, (request: HTTPRequest) => {
```

Comment:
Is type here for property still needed?

Reply:
Please explain why we still need this.

Reply:
It looks like we don't need this anymore can we put TODO/Issue to remove them and rely on our Types.